src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+  background-color: #000;
+  padding: 20px;
+}
+
+.block,
+.block-smaller,
+.block-smallest {
+  border-width: 1px;
+  border-style: solid;
+  border-color: #e75551;
```

Comment:
`border: 1px solid #e75551` will be better

Reply:
ok, thanks very much. Solved